bot/exts/info/pypi.py

```diff
@@ -67,8 +69,14 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     log.trace(f"Error when fetching PyPi package: {response.status}.")
 
         if error:
-            await ctx.send(embed=embed, delete_after=INVALID_INPUT_DELETE_DELAY)
-            await ctx.message.delete(delay=INVALID_INPUT_DELETE_DELAY)
+            error_message = await ctx.send(embed=embed)
+            await wait_for_deletion(error_message, (ctx.author.id,), timeout=INVALID_INPUT_DELETE_DELAY)
+
+            # If won't ghost-ping when deleting message
```

Comment:
This comment is still very weirdly worded.

Reply:
Forgot to reword it when pasting over. Will do that